python/api/host/serializers.py

```diff
@@ -31,9 +31,10 @@ class HostSerializer(serializers.Serializer):
     prototype_id = serializers.IntegerField(help_text='id of host type')
     provider_id = serializers.IntegerField()
     fqdn = serializers.CharField(help_text='fully qualified domain name')
-    description = serializers.CharField(required=False)
+    description = serializers.CharField(required=False, allow_blank=True)
     state = serializers.CharField(read_only=True)
     url = ObjectURL(read_only=True, view_name='host-details')
+    maintenance_mode = serializers.ChoiceField(choices=MaintenanceModeType.choices, required=True)
```

Comment:
I think `required=True` is redundant. We want this field not to be required when creating an object, but only required when updating